docs/ref/contrib/admin/index.txt

```diff
@@ -2505,10 +2505,13 @@ Templates can override or extend base admin templates as described in
     * ``site_title``: :attr:`AdminSite.site_title`
     * ``site_url``: :attr:`AdminSite.site_url`
     * ``has_permission``: :meth:`AdminSite.has_permission`
+    * ``available_apps``: a list of applications from the
+        :ref:`application registry</ref/applications/>` available for the current
```

Comment:
should be doc instead of ref
also a line break is allowed inside that reference between words so you can make the previous line a bit longer